src/main/webapp/WEB-INF/views/developerTable.jsp

```diff
@@ -0,0 +1,29 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<!DOCTYPE html>
+<html lang="en">
+<head></head>
+<center>
+<body>
+    <h2>Developers Table</h2>
+    <table>
+        <tr>
+            <th>ID</th>
```

Comment:
It is not good to hard code this values.
They should be retrieved from the backend.

Reply:
fixed